pkg/observer/observer.go

```diff
@@ -149,8 +150,9 @@ func (k *Observer) getRBQueueSize() int {
 // notified of their corresponding events.
 type Observer struct {
 	/* Configuration */
-	listeners  map[Listener]struct{}
-	PerfConfig *bpf.PerfEventConfig
+	listeners      map[Listener]struct{}
+	PerfConfig     *bpf.PerfEventConfig
+	RingBufMapName string
```

Comment:
qq, have we given any thought to multiple rings per cpu? Definitely not for this PR, just curious. There may be a benefit to doing it, if we haven't checked yet. cc @kkourt

Reply:
Perhaps is due to some later kernel patch involving oom score computation? I remember i saw something about it; let me look it up!